docs/examples.md

```diff
@@ -246,4 +246,5 @@ data-validation validate custom-query --custom-query-type column --source-query-
 data-validation validate custom-query --custom-query-type row --source-query-file source_query.sql --target-query-file target_query.sql -sc my_bq_conn -tc my_bq_conn -tbls bigquery-public-data.new_york_citibike.citibike_stations  --hash \'*\'
 ````
 
-Please replace source_query.sql and target_query.sql with the correct files containing sql query for source and target database respectively.
\ No newline at end of file
+Please replace source_query.sql and target_query.sql with the correct files containing sql query for source and target database respectively. The primary key should be included
+in the SELECT statement of both source_query.sql and target_query.sql. Also, the --primary-key flag is a required arguement.
```

Comment:
Let's include the `--primary-key` flag in the CLI description [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/README.md?plain=1#L302).